src/Themes/VuetifyTheme/Views/Content-VList.liquid

```diff
@@ -0,0 +1,16 @@
+<v-list>
+  {% for item in Model.Content.ListPart.ContentItems %}
+      {{ item | shape_build_display | shape_render }}
+      <v-divider inset></v-divider>
+  {% endfor %}
+</v-list>
+
+{% assign previousText = "New" | t %}
```

Comment:
The pager stuff isn't needed anymore. I think only ListParts support pagination.